runtime/ztests/expr/function/split.yaml

```
@@ -5,6 +5,7 @@ vector: true
 input: |
   {s:"a,b,c",sep:","((int64,string))}
   {s:"abc",sep:""}
+  {s:"a",sep:","}
```

runtime/sam/expr/ztests/split.yaml also includes an empty-string case so I think we should add that here.
```suggestion
  {s:"",sep:","}
```